integrations/cassandra/src/cassandra_test.go

```diff
@@ -28,29 +30,19 @@ func TestPopulatelMetrics(t *testing.T) {
 		"functionSource": {functionSource, metric.GAUGE},
 	}
 
-	var sample = metric.NewMetricSet("eventType")
-	populateMetrics(&sample, rawMetrics, metricDefinition)
+	s, err := metric.NewSet("eventType", persist.NewInMemoryStore())
```

Comment:
Why here we use different function than in the `main`? `e.NewMetricSet`

Reply:
NewMetricSet was renamed to NewSet on the SDKv3